docs/Changelog.md

```diff
@@ -2960,7 +2969,16 @@ opset_import {
    the tensor according to kernel sizes, stride sizes, and pad lengths.
    max pooling consisting of computing the max on all values of a
    subset of the input tensor according to the kernel size and downsampling the
-   data into the output tensor Y for further processing.
+   data into the output tensor Y for further processing. The output spatial shape will be following:
+   ```
+   VALID or pads given: output_spatial_shape[i] = ceil((input_spatial_shape[i] - (kernel_spatial_shape[i] - 1)) / strides_spatial_shape[i]))
+   SAME_UPPER or SAME_LOWER: output_spatial_shape[i] = ceil(input_spatial_shape[i] / strides_spatial_shape[i]))
```

Comment:
Please notice that: auto_pad is a deprecated attribute, we encourage users to use explicit padding.

Please also add how we calculate the output_shape, when we use explicit padding:
Some reference is here:
http://pytorch.org/docs/master/nn.html#torch.nn.AvgPool2d
I think most frameworks compute the output shape using this formula. (Checked Caffe2, PyTorch, MXNet)

Reply:
@houseroad Actually, two formulas are same. ( I just ignored I should add pad_spatial to input_spatial_shape.) Anyway, I will fix it.